graphstorm-processing/docker/0.2.2/sagemaker/Dockerfile.cpu

```diff
@@ -11,32 +11,17 @@ ENV LC_ALL=C.UTF-8
 ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
 ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/conda/lib"
 ENV PATH=/opt/conda/bin:$PATH
+ENV PIP_NO_CACHE_DIR=1
 
-# Install GSProcessing requirements to pipenv Python
-RUN pipenv install \
```

Comment:
Compared with the previous pipenv and current pip install options, the current one ran 1 hour on amazon review, but previous one ran 40 minutes on the amazon review data processing. Will pipenv here do something different to have that influence?

Reply:
Very unlikely, but not impossible (at this point I've seen everything). Let me try with some reference datasets for which I know the expected runtime and see if it makes a consistent difference.

Reply:
Test by doing 2 runs for 2 jobs, comparing runtimes with these changes vs. previous version of the image. The runtimes are very close (+/- 5% which is expected for SM). 

We can take a look at replicating this particular issue for this dataset and see if the difference is persistent (could be for example including MKL or other env difference). For this PR though I'd consider this a non-issue.